src/assets/formatted/AccountCircle.tsx

```diff
@@ -0,0 +1,47 @@
+import { Ref, forwardRef, memo } from 'react';
+import { kebabCase } from 'change-case';
+
+interface SvgAccountCircleProps {
+  color?: string;
+  height?: number | string;
+  width?: number | string;
+}
+
+const SvgAccountCircle = memo(
+  forwardRef((props: SvgAccountCircleProps, ref: Ref<SVGSVGElement>) => {
+    const { color, height, width } = props;
+    const title = 'SvgAccountCircle';
```

Comment:
I was hoping the consumer could set the `title` as a prop since they'll need to translate it in certain cases.  The titleId might need to be passed as well as an `id` since there might be multiple account circles on the same page.